### PR TITLE
fix typo in function name (form -> from)

### PR DESCRIPTION
**Pull request by Be-ing**

### CODE REVIEW

File: src/window_identifier.rs [hunk 1]
```diff
@@ -285,7 +285,7 @@ impl WindowIdentifier {
     /// Create an instance of [`WindowIdentifier`] from a [`RawWindowHandle`](raw_window_handle::RawWindowHandle).
     ///
     /// The constructor returns a valid handle under both Wayland & x11.
-    pub fn form_raw_handle(handle: &RawWindowHandle) -> Self {
```
Comment (bilelmoussaoui) [inline src/window_identifier.rs:288]: Oh no

**Comment by bilelmoussaoui** [comment]
I will merge this once the CI passes and do 0.2.1 release with that

**Comment by bilelmoussaoui** [comment]
0.2.1 should be out

**Comment by Be-ing** [comment]
Thanks. This is technically a breaking change, but I am quite doubtful anyone is actually depending on this typo considering it was released just yesterday and anyone trying to use it would have noticed the typo.

**Comment by bilelmoussaoui** [comment]
Yes I'm aware that is a breaking change but it is a part of the API that is very unlikely to be used by the most common users of ashpd which are applications that is why I didn't want to bump to 0.3 just for that :)

**Comment by bilelmoussaoui** [comment]
That and also being behind a feature flag

**Pull request merged**